### PR TITLE
Added Void Linux package name to README

### PR DESCRIPTION
**Pull request by notramo**

### CODE REVIEW

**Comment by jbeich** [comment]
Why not always up-to-date badge instead?
[Packaging status](https://repology.org/project/swappy/versions)

**Comment by jtheoof** [comment]
@jbeich would be happy to accept MR for that one. @notramo. The commint lint still does not pass. You see why in the logs.

**Comment by itspngu** [comment]
Not meaning to hijack this discussion, but would you be willing to look into providing binary builds for some more distros @jtheoof? 

I could try and help with setting things up if that'd help, meant to look into Travis CI for a while anyway and got a vacation (+ covid lockdown) coming up ;)

**Comment by notramo** [comment]
@jtheoof: The problem is that the commit message starts with capital letter. I can't force-push from the command line to my fork, and the web UI don't have an option to force-push.
Wouldn't it be easier if you added that simple line from your account?

File: README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ text_font=sans-serif
 - [Arch Linux (git)](https://aur.archlinux.org/packages/swappy-git)
 - [Fedora 31/32](https://copr.fedorainfracloud.org/coprs/wef/swappy)
 - [openSUSE](https://build.opensuse.org/package/show/X11:Wayland/swappy)
+- Void Linux: the `swappy` package
```
Comment (itspngu) [inline README.md:103]:
```suggestion
- [Void Linux](https://github.com/void-linux/void-packages/tree/master/srcpkgs/swappy)
```

**Review by itspngu — CHANGES_REQUESTED** [review]
> The problem is that the commit message starts with capital letter. I can't force-push from the command line to my fork, because GitHub doesn't allow it, and the web UI don't have an option to force-push.
Wouldn't it be easier if you added that simple line from your account?

I think it'd be easiest if you made a new branch, added a link to the void-sources repo tree as shown (so it's in line with the other distros), paid attention to formatting the commit message as needed by `commitlint`, and then submitted a new PR. My 2 cents. Cheers

**Comment by jtheoof** [comment]
Thanks @itspngu @notramo. I've added the line in the README in [this commit](https://github.com/jtheoof/swappy/commit/a7fda1f998798811b54c8ee9dbfc147f43b97e96). I'll close this PR.

**Pull request closed without merging**